src/Uno.UWP/System/Launcher.cs

```diff
@@ -25,6 +25,7 @@ public static Task<bool> LaunchUriAsync(Uri uri)
 				throw new ArgumentNullException(nameof(uri));
 			}
 
+#if !__WASM__
```

Comment:
For this type of case there's also `CoreDispatcher.CheckThreadAccess()` which is already disabled for WASM. (The advantage being having less places in code that need to be changed when threading becomes available)